src/test/components/PublicRoute.spec.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { PublicRoute } from '../../components/PublicRoute/PublicRoute';
+
+const props = {
+  component: () => {},
+  isAuthenticated: false,
+};
+describe('component: PraveRoute', () => {
```

Comment:
Shouldn't this be `PublicRoute` in the describe
`describe.only('component: PublicRoute', () => {`